### ISSUE TITLE
Does not work with closed wiki

### ISSUE THREAD

**Issue by flesser** [Schine/MW-OAuth2Client#4]
If you have a closed wiki (without public read access, i.e. `$wgGroupPermissions['*']['read'] = false;`), login via this extension is not possible because the page `Special:OAuth2Client/redirect` is not accessible for not logged in users...

**Comment by wgmv** [comment]
Have you added `Special:OAuth2Client/redirect` to $wgWhitelistRead = []?

**Comment by f3ndot** [comment]
Can confirm I've been able to get it working on a private/closed wiki (MediaWiki 1.33.0, extension 0.3) with the following settings in `LocalSettings.php`

```php
$wgWhitelistRead = ['Special:OAuth2Client', 'Special:OAuth2Client/redirect'];
```

I think this issue can be closed.
